docker-compose-dev.yml

```diff
@@ -209,7 +209,7 @@ services:
       - flow-api
 
   minio:
-    image: minio/minio:RELEASE.2024-07-16T23-46-41Z
+    image: minio/minio:RELEASE.2024-11-07T00-52-20Z
```

Comment:
can swap these to the bitnami image, or as a followup

Reply:
would prefer as followup, these images work well enough as is, but would be nice to have docker-compose depend on same images as CI